.changelog/v0.9.0/bug-fixes/ibc/1532-connOpenAck-counterparty-conn-id-not-set.md

```diff
@@ -0,0 +1,5 @@
+- Set the connection counterparty in the ICS03 [`connOpenAck` handler][conn-open-ack-handler]
```

Comment:
Wondering if we should continue to keep these files (`.changelog/*`) in the repo or just delete them before every release?

Reply:
You might decide to edit changelog entries retroactively and reassemble `CHANGELOG.md` from them.
Then again (and I haven't yet checked if `unclog` appends incrementally only the entries from `.changelog/unreleased`) it might just be easier to fix up the stable part of `CHANGELOG.md` directly.

Reply:
Right... Currently, we use the `unclog build -u` to build only the unreleased changes and prepend that to the CHANGELOG.md (https://github.com/informalsystems/ibc-rs/blob/master/CONTRIBUTING.md#changelog).

Reply:
Generally, it makes sense to reduce copies of the same content that are checked into version control, the idea being that there should be one "source of truth" that is checked in. Once the files in `.changelog/unreleased` have served their purpose of deconflicting pending PRs and their content has been synced into `CHANGELOG.md`, they can be removed.

Reply:
>Once the files in .changelog/unreleased have served their purpose of deconflicting pending PRs and their content has been synced into CHANGELOG.md, they can be removed.

+1